packages/jobs/src/core/consts.ts

```diff
@@ -0,0 +1,14 @@
+import console from 'node:console'
+
+export const DEFAULT_MAX_ATTEMPTS = 24
+export const DEFAULT_MAX_RUNTIME = 14_400 // 4 hours in seconds
```

Comment:
```suggestion
/** 4 hours in seconds */
export const DEFAULT_MAX_RUNTIME = 14_400
```

A doc comment is better here because then VSCode will show it whenever you hover the variable in other places/files in the code